components/script/textinput.rs

```diff
@@ -137,28 +232,30 @@ pub const CMD_OR_CONTROL: Modifiers = Modifiers::META;
 #[cfg(not(target_os = "macos"))]
 pub const CMD_OR_CONTROL: Modifiers = Modifiers::CONTROL;
 
+// FIXME this function does not behave as described (if given string has fewer than n
```

Comment:
happy to change this as a drive-by. Wasn't sure if I should be affected end behavior for this issue or not.

Reply:
Sorry I wasn’t clear by commenting about this here rather than at the top level. Avoiding to modify the existing commit was intended for any change after the first review pass, not just this one.

Reply:
So I should revert the fix for now and add it back in once this commit has been approved? Or maybe an entirely different PR?